roles/satosa/tasks/main.yml

```diff
@@ -260,18 +268,6 @@
       args:
         chdir: "{{ satosa_env_dir }}"
 
-    - name: Retrieve Test SP metadata
-      get_url:
-        url: https://{{ hostnames.sp }}/saml/module.php/saml/sp/metadata.php/default-sp
-        dest: "{{ satosa_env_dir }}/metadata/sp-test.xml"
-        force: yes
-
-    - name: Retreive Test IdP metadata
-      get_url:
-        url: https://{{ hostnames.idp }}/saml/saml2/idp/metadata.php
-        dest: "{{ satosa_env_dir }}/metadata/idp-test.xml"
-        force: yes
-
```

Comment:
Why are you removing these?

Reply:
Because we get sp-test and idp-test metadata from pyff. Testing IgnorePolicy was nog possible with the local metadata files in place and they should be removed anyway.

Reply:
Disadvantage is that we now don't have correct mirrored metadata, but it's a mess anyway ;)